Fixes And Cleans Up Manuals

Zero-Bits opened the pull request:
All the manuals are now under one file, instead of mostly in one file with 3-4 scattered around other files arbitrarily. In addition, all the manuals now point to the correct wiki pages.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

comma commented:
Pretty sure torch specific one is intended to be in maps folder, kinda against the spirit of map system to put it back into global space.

Zero-Bits commented:
I would tend to agree, but there were only two items in there, and the rest of the Torch-related manuals (SCMJ, SCG Law, etc.) were in the main manuals file. That said, I could make a commit that moves all manuals relevant to the Torch's setting to the Torch manuals file if that would be preferred.

Zero-Bits commented:
Actually, the manuals file is messy enough as it is, I have a better idea. Which includes the previous idea. So not really a better idea, just an additional idea.

Zero-Bits committed:
Fixes And Cleans Up Manuals

The manual files are now organized in a much more scalable way. In
addition, the broken manuals have been pointed to the right wiki page,
and the Space Law manual has been removed due to it pointing to the same
page as the NT Regs manual anyway.

Zero-Bits commented:
The manual files are now organized in a much more scalable way. In addition, the broken manuals have been pointed to the right wiki page, and the Space Law manual has been removed due to it pointing to the same page as the NT Regs manual anyway.

This is way better than just sticking all of them in one file, especially if anyone wants to add more manual-type books, or is looking for the code for a specific manual in order to edit/remove it. A lot more convenient, and a lot more scalable than before.

PsiOmegaDelta commented:
Won't these books become outdated faster than the Wiki ever was?

Zero-Bits commented:
All of these manuals already existed, I haven't made new ones or changed the actual content of non-wiki ones, just cleaned up the file-structure and updated all the manuals that link to the wiki (which will always be exactly as outdated as the wiki)

I'm /hoping/ to eventually move all manuals to the wiki so it's easier to update them, but, for the time being, and while the wiki is still being figured out, that's out of my current scope.
